photon-core/src/test/java/org/photonvision/vision/pipeline/UncropApriltagTest.java

```diff
@@ -0,0 +1,108 @@
+        var pipeline = new AprilTagPipeline();
+
+        pipeline.getSettings().inputShouldShow = true;
+        pipeline.getSettings().outputShouldDraw = true;
+        pipeline.getSettings().solvePNPEnabled = true;
+        pipeline.getSettings().cornerDetectionAccuracyPercentage = 4;
+        pipeline.getSettings().cornerDetectionUseConvexHulls = true;
+        pipeline.getSettings().targetModel = TargetModel.kAprilTag6p5in_36h11;
+        pipeline.getSettings().tagFamily = AprilTagFamily.kTag36h11;
+
+        var frameProvider =
+                new FileFrameProvider(
+                        TestUtils.getApriltagImagePath(TestUtils.ApriltagTestImages.kTag1_640_480, false),
+                        TestUtils.WPI2020Image.FOV,
+                        TestUtils.get2020LifeCamCoeffs(false));
+        frameProvider.requestFrameThresholdType(pipeline.getThresholdType());
+
+        CVPipelineResult pipelineResult;
+        pipelineResult = pipeline.run(frameProvider.get(), QuirkyCamera.DefaultCamera);
+
+        printTestResults(pipelineResult);
+
+        // Draw on input
+        var outputPipe = new OutputStreamPipeline();
+        var ret =
+                outputPipe.process(
+                        pipelineResult.inputAndOutputFrame, pipeline.getSettings(), pipelineResult.targets);
+
+        TestUtils.showImage(ret.inputAndOutputFrame.processedImage.getMat(), "Pipeline output", 999999);
+
+        pipeline.getSettings().static_x = 100;
+
+        CVPipelineResult croppedResults;
+        croppedResults = pipeline.run(frameProvider.get(), QuirkyCamera.DefaultCamera);
+
+        printTestResults(croppedResults);
+
+        var ret_cropped =
+                outputPipe.process(
+                        croppedResults.inputAndOutputFrame, pipeline.getSettings(), croppedResults.targets);
+
+        TestUtils.showImage(
+                ret_cropped.inputAndOutputFrame.processedImage.getMat(), "Cropped Pipeline output", 999999);
+        // these numbers are not *accurate*, but they are known and expected
+        var target = pipelineResult.targets.get(0);
+        var croppedTarget = croppedResults.targets.get(0);
+
+        // Test corner order
+        var corners = target.getTargetCorners();
+        var croppedCorners = croppedTarget.getTargetCorners();
+
+        Assertions.assertEquals(corners.get(0).x, croppedCorners.get(0).x, 0.1);
+        Assertions.assertEquals(corners.get(0).y, croppedCorners.get(0).y, 0.1);
+        Assertions.assertEquals(corners.get(1).x, croppedCorners.get(1).x, 0.1);
+        Assertions.assertEquals(corners.get(1).y, croppedCorners.get(1).y, 0.1);
+        Assertions.assertEquals(corners.get(2).x, croppedCorners.get(2).x, 0.1);
+        Assertions.assertEquals(corners.get(2).y, croppedCorners.get(2).y, 0.1);
+        Assertions.assertEquals(corners.get(3).x, croppedCorners.get(3).x, 0.1);
+        Assertions.assertEquals(corners.get(3).y, croppedCorners.get(3).y, 0.1);
+
```

Comment:
Consider replacing with https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#assertIterableEquals-java.lang.Iterable-java.lang.Iterable- since Point overrides equals()